utilities/verifySupportUtility/go/README.md

```diff
@@ -0,0 +1,118 @@
+# IBM Security Performance
+
+## Verify - Identity and Access Management
+
+### Verify examples
+
+#### Verify Support Utility
+
+This directory contains example code for uploading and listing files for use by Verify Support.
+
+### Table of Contents
+
+* [Usage of the example](#usage-of-the-example)
+* [Scenarios/User Stories](#scenariosuser-stories)
+* [Building the example](#building-the-example)
+
+### Usage of the example
+```text
+Usage: verifySupportUtility [auth|status|upload|list|results|help] -tenantURL tenantURL [-uploadFile filename] [-uploadComment comment] [-resultsFile filename]
+Usage of verifySupportUtility:
+        command is one of [ auth, status, upload, list, results, help ]
+        (default is 'help')
+  -help
+        Display the full help text
+  -loglevel integer
+        Logging Level (default 0)
+        0=report success/failure, status codes, response times  (default)
+        1=report include response body
+        2=report include request body
+        3=report full trace (for debugging)
+  -tenantURL tenantURL
+        URL used to contact tenant: client:secret@tenant.domain
+  -uploadFile filename
+        Optional file to upload to Verify
+  -uploadComment comment
+        Optional comment to include with upload to Verify
+  -resultsFile filename
+        Optional results file to download from Verify
+
+auth	Authenticate to the tenant using the specified client and secret
+status	Query status of the file upload service
+upload	Upload the specified uploadFile to Verify
+list	List uploaded files
+results	List any results or download a results file
+help	Display the full help text
+```
+
+
+### Scenarios/User Stories
+
+* Customer wishes to provide a large file to SRE via secure channel.
+* SRE directs customer to use the Verify support utility with their tenant.
+* Customer creates an API client, or uses an existing one, in their tenant.
```

Comment:
No, at the moment as long as you have authenticated with a client for that tenant it will accept the uploads.
We have the option of adding an entitlement in the future if we decide we need to, but that adds more step before customers can upload a file to support.